### PR TITLE
Update AndroidKeyCode.java

### PR DESCRIPTION
**Pull request by fireashes**
Added all the Key Event from http://developer.android.com/reference/android/view/KeyEvent.html

### CODE REVIEW

**Comment by TikhomirovSergey** [comment]
:+1:

File: src/main/java/io/appium/java_client/android/AndroidKeyCode.java [hunk 1]
```diff
@@ -23,12 +23,1273 @@
...
 
   int BACK = 4;
-  int BACKSPACE = 67;
-  int DEL = 67;
-  int ENTER = 66;
-  int HOME = 3;
-  int MENU = 82;
-  int SETTINGS = 176;
-  int SPACE = 62;
+    int BACKSPACE = 67;
```
Comment (moizjv) [inline src/main/java/io/appium/java_client/android/AndroidKeyCode.java]: indentation

File: src/main/java/io/appium/java_client/android/AndroidKeyCode.java [hunk 2]
```diff
@@ -23,12 +23,1273 @@
...
+    int ENTER = 66;
+    int HOME = 3;
+    int MENU = 82;
+    int SETTINGS = 176;
+    int SPACE = 62;
+
+
+    int ACTION_DOWN = 0;
+    //getAction() value: the key has been pressed down.
+    //(0x00000000)
```
Comment (moizjv) [inline src/main/java/io/appium/java_client/android/AndroidKeyCode.java]: Comments on top instead on bottom. 

```
// getAction() value: the key has been pressed down.
// (0x00000000)
int ACTION_DOWN = 0;
```

File: src/main/java/io/appium/java_client/android/AndroidKeyCode.java [hunk 3]
```diff
@@ -23,12 +23,1273 @@
...
+    int ACTION_MULTIPLE = 2;
+    //getAction() value: multiple duplicate key events have occurred in a row, or a complex string is being delivered. If the key code is not {#link KEYCODE_UNKNOWN then the {#link getRepeatCount() method returns the number of times the given key code should be executed. Otherwise, if the key code is KEYCODE_UNKNOWN, then this is a sequence of characters as returned by getCharacters().
+    //(0x00000002)
+
+    int ACTION_UP = 1;
+    //getAction() value: the key has been released.
+    //(0x00000001)
+
+    int FLAG_CANCELED = 32;
+    //When associated with up key events, this indicates that the key press has been canceled. Typically this is used with virtual touch screen keys, where the user can slide from the virtual key area on to the display: in that case, the application will receive a canceled up event and should not perform the action normally associated with the key. Note that for this to work, the application can not perform an action for a key until it receives an up or the long press timeout has expired.
```
Comment (moizjv) [inline src/main/java/io/appium/java_client/android/AndroidKeyCode.java]: space after `//`

like

```
// When associated with up key events, this
```

**Comment by Jonahss** [comment]
I'll make the corrections

**Comment by Jonahss** [comment]
Merged patch-2 as #207 

Sorry for the delay @fireashes thanks for your help.

**Pull request closed without merging**